homeassistant/components/decora/light.py

```diff
@@ -1,20 +1,23 @@
 """Support for Decora dimmers."""
+from functools import wraps
 import importlib
 import logging
-from functools import wraps
 import time
 
+from bluepy.btle import BTLEException  # pylint: disable=import-error, no-member
 import voluptuous as vol
 
-from homeassistant.const import CONF_API_KEY, CONF_DEVICES, CONF_NAME
 from homeassistant.components.light import (
     ATTR_BRIGHTNESS,
+    PLATFORM_SCHEMA,
     SUPPORT_BRIGHTNESS,
     Light,
-    PLATFORM_SCHEMA,
 )
+from homeassistant.const import CONF_API_KEY, CONF_DEVICES, CONF_NAME
 import homeassistant.helpers.config_validation as cv
 
+DECORA = importlib.import_module("decora")
```

Comment:
Is [that] how the usage of `importlib` is supposed to be moved!?
pylint is happy and isort as well...

Reply:
This isn't an import.  Don't let the name of the module confuse you.  :)  This is calling the function `import_module` of the module `importlib`.

Reply:
In looking at this code, should it just be changed to a standard import statement?  Like `import decora`  or more specifically, `from decora import decora as decora_device` and update the reference below.

Reply:
Yeah this should be `import decora`

Reply:
Isn't it done that way because the module has the same name as the one that gets imported? 
I saw a commit message for the bme680 component that said it was done because of that over there.

Reply:
Yeah but that's janky.  The proper way to do it is to alias it if it collides (use `as`)

Reply:
that used to be the case when it was `sensor/decora.py`. That is no longer the case.